src/main/java/com/educare/unitylend/controller/WalletController.java

```diff
@@ -31,48 +31,79 @@ public class WalletController extends BaseController {
     public List<Wallet> getAllWallets() throws ControllerException {
         try {
             List<Wallet> walletList = walletService.getWallets();
+
             return walletList;
         } catch (Exception e) {
             log.error("Error encountered in getting the users");
             throw new ControllerException("Error encountered in getting the users", e);
         }
     }
 
-    @GetMapping("/get-user-info/{userId}")
-    public Wallet getWalletInfo(@PathVariable String userId) throws ControllerException{
 
+    @GetMapping("/get-wallet-info/{userId}")
+    public ResponseEntity<Wallet> getWalletInfo(@PathVariable String userId) throws ControllerException {
```

Comment:
Can u write the function name getWalletInfoByUserId, like it would help us in future , because there are 2 apis for getting wallet info , one by user id and another by wallet id. It may create confusion later.